pupil_src/launchables/player.py

```diff
@@ -382,7 +380,8 @@ def set_window_size():
                            ('Vis_Circle', {}),
                            ('System_Graphs', {}),
                            ('System_Timelines', {}),
-                           ('Video_Export_Launcher', {}),
+                           ('World_Video_Exporter', {}),
+                           ('Eye_Video_Exporter', {}),
```

Comment:
@mkassner What do you think, should we add the eye video exporter as default plugin?